endpoints/endpoints.go

```diff
@@ -106,14 +122,18 @@ func Store(ctx context.Context, rc *rest.Config, ns string, ep Endpoint) error {
 }
 
 const (
-	clientCertLabel = "client-certificate-data"
-	clientKeyLabel  = "client-key-data"
-	caLabel         = "certificate-authority-data"
-	proxyUrlLabel   = "proxy-url"
-	serverUrlLabel  = "server-url"
-	debugLabel      = "debug"
-	passwordLabel   = "password"
-	usernameLabel   = "username"
-	tokenLabel      = "token"
-	insecureLabel   = "insecure"
+	clientCertLabel   = "client-certificate-data"
+	clientKeyLabel    = "client-key-data"
+	caLabel           = "certificate-authority-data"
+	proxyUrlLabel     = "proxy-url"
+	serverUrlLabel    = "server-url"
+	debugLabel        = "debug"
+	passwordLabel     = "password"
+	usernameLabel     = "username"
+	tokenLabel        = "token"
+	insecureLabel     = "insecure"
+	awsAccessKeyLabel = "awsAccessKey"
```

Comment:
Just a note: is there a reason some JSON keys use kebab-case while others use camelCase? It’s not a big issue, but using a single style would improve consistency.